.flowconfig

```diff
@@ -8,4 +9,5 @@
 
 [options]
 module.name_mapper='.*\(\.s?css\)' -> 'empty/object'
+module.name_mapper='^\(\(components\|containers\|fixtures\|middleware\|routes\|styleguide\|styles\|util\)/.*\)$' -> '<PROJECT_ROOT>/src/\1'
```

Comment:
This is annoying, but the only way I can find to get Flow to recognize the relative module imports...